server/api/users/users.test.ts

```diff
@@ -36,21 +37,108 @@ describe('auth router', () => {
     });
   });
 
-  describe('GET /', () => {
-    it('expects empty array if no users exist', async () => {
+  describe('GET /api/users', () => {
+    it('should have an API endpoint that exists', async () => {
       const res = await server.exec.get('/api/users/');
-      expect(res.body).toStrictEqual([]);
+
+      expect(res.status).toBe(200);
+    });
+
+    it('should return empty array if no users exist', async () => {
```

Comment:
i think it would be nice to easily distinguish between happy path and unhappy path test cases by having unhappy cases start with "should error" and happy cases not.  So this one might instead be:

```
"should error with "empty array if no users exist"'
```

Reply:
Why should it error when it's returning a 200 status code?

Reply:
oops - meant to leave this comment on the next one that actually errors 🤡